### PR TITLE
Added TeleportWarmupCancelledEvent

### PR DESCRIPTION
**Pull request by alexdev03**

### CODE REVIEW

File: common/src/main/java/net/william278/huskhomes/event/ITeleportWarmupCancelledEvent.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    /**
+     * The reason the teleport warmup was cancelled.
+     *
+     * @return the reason the teleport warmup was cancelled
+     */
+    @NotNull
+    CancelReason getCancelReason();
+
+    enum CancelReason {
```
Comment (WiIIiam278) [inline common/src/main/java/net/william278/huskhomes/event/ITeleportWarmupCancelledEvent.java:60]: You'll want to make this enum `public` - and perhaps some JavaDoc comments?

File: docs/API-Events.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+| `WarpListEvent`                |  3.0  |      ✅      | Called when a player requests to view a list of warps                    |
+| `WarpEditEvent`                |  4.0  |      ✅      | Called when a player edits a warp (relocate, description, name)          |
+| `WarpDeleteEvent`              |  3.0  |      ✅      | Called when a player deletes a warp&dagger;                              |
+| `DeleteAllWarpsEvent`          | 3.2.1 |      ✅      | Called when a player uses `/delwarp all` to delete all warps             |
+| `SendTeleportRequestEvent`     |  4.1  |      ✅      | Called when a player sends a teleport request (`/tpa`)                   |
+| `ReceiveTeleportRequestEvent`  |  4.1  |      ✅      | Called when a player receives a teleport request from someone            |
+| `ReplyTeleportRequestEvent`    |  4.1  |      ✅      | Called when a player accepts or declines a teleport request              |
+| `TeleportWarmupEvent`          |  3.0  |      ✅      | Called when a player starts a teleport warmup countdown                  |
+| `TeleportWarmupCancelledEvent` | 4.6.3 |      ✅      | Called when a player cancels the teleport warmup                         |
```
Comment (WiIIiam278) [inline docs/API-Events.md:20]: This event is not cancellable, so we'll need an :x: here.

**Review by WiIIiam278 — CHANGES_REQUESTED** [review]
Some tiny changes, looks great otherwise

File: common/src/main/java/net/william278/huskhomes/event/ITeleportWarmupCancelledEvent.java (hunk 1 above)
Comment (alexdev03) [reply common/src/main/java/net/william278/huskhomes/event/ITeleportWarmupCancelledEvent.java:60]: public is useless in interfaces

**Review by WiIIiam278 — APPROVED** [review]

**Pull request merged**